src/styles/_catalog.scss

```diff
@@ -1,22 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
-    height: 300px;
+    width: $cardWidth;
+    height: $cardHeight;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
+}
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+@each $columnAmount, $screenWidth in $screen {
+  @media (min-width: $screenWidth) {
+    .catalog {
+      grid-template-columns: repeat($columnAmount, $cardWidth);
+    }
   }
 }
```

Comment:
🔥

Reply:
also you don't need to use a selector in your media query, SASS nesting allows you to make a media query right in your .catalog selector